sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -3079,6 +3083,48 @@ public boolean isCaseEditAllowed(String caseUuid) {
 		return caseService.isCaseEditAllowed(caze);
 	}
 
+	@Override
+	public void sendMessage(List<String> caseUuids, String subject, String messageContent, MessageType... messageTypes) {
+		caseUuids.forEach(uuid -> {
+			final Case aCase = caseService.getByUuid(uuid);
+			final Person person = aCase.getPerson();
+
+			try {
+				messagingService.sendMessage(person, subject, messageContent, messageTypes);
+			} catch (NotificationDeliveryFailedException e) {
+				logger.error(
+					String.format(
+						"NotificationDeliveryFailedException when trying to notify person about: %s" + "Failed to send " + e.getMessageType()
+							+ " to person with UUID %s.",
+						subject,
+						person.getUuid()));
+			}
+		});
+	}
+
+	@Override
+	public long countCasesWithMissingMessageType(List<String> caseUuids, MessageType messageType) {
```

Comment:
* This method will fail if it's called for more than 32,000 cases; use IterableHelper to execute this batched
* Rename the method to "countCasesWithMissingContactInformation" and add a comment to explain what it's supposed to do

Reply:
is it a realistic scenario that the UI will send 32K + selected caseUuids ?
i can do batch counting but that feel really odd to me

Reply:
It's theoretically possible because the bulk edit view is not lazy (since that doesn't work with "select all"); I agree it's not that realistic, but it's a potential bug that we should rule out nonetheless I think